app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,50 @@
+class ReservationsController < ApplicationController
+  before_action :set_reservation, only: %i[show edit update]
+  before_action :set_screenings, only: %i[new edit]
+
+  def index
+    @reservations = Reservation.all.includes(screening: :movie)
```

Comment:
you don't need the `.all`
```suggestion
    @reservations = Reservation.includes(screening: :movie)
```